.github/actions/setup/action.yml

```diff
@@ -3,26 +3,36 @@ name: Setup environment
 inputs:
   cache:
     description: Enable caching
-    default: "true"
+    default: 'true'
   node:
     description: The Node.js version to install
     required: true
   solana:
     description: The Solana version to install
 
 runs:
-  using: "composite"
+  using: 'composite'
   steps:
     - name: Setup pnpm
       uses: pnpm/action-setup@v3
     - name: Setup Node.js
       uses: actions/setup-node@v4
       with:
         node-version: ${{ inputs.node }}
-        cache: "pnpm"
+        cache: 'pnpm'
     - name: Install dependencies
       run: pnpm install --frozen-lockfile
       shell: bash
+    - name: Install Rust
+      uses: metaplex-foundation/actions/install-rust@v1
```

Comment:
Is there a reason to use this action over https://github.com/dtolnay/rust-toolchain ?

Reply:
It's using that action underneath, so I just went with Metaplex since we're already using it for Solana.

Reply:
Let's remove the layer of indirection if it isn't needed